esp/scm/ws_dali.ecm

```diff
@@ -45,13 +45,31 @@ ESPrequest [nil_remove] DeleteRequest
     string Path;
 };
 
+ESPrequest [nil_remove] AddRequest
+{
+    string Path;
+    string Value;
+};
+
+ESPrequest [nil_remove] CountRequest
+{
+    string Path;
+};
+
+ESPresponse [exceptions_inline, nil_remove] CountResponse
+{
+    unsigned Result;
```

Comment:
minor: is it worth a new resopnse type, or could you return a string and re-use ResultResponse ?
What are the benefits/disadvantages ?

Reply:
@jakesmith A user may be confused if the Result is defined as string (why a count is a string?). Another benefit is that, if the Result is used for calculation or looping, the caller does not need to cast it from a string to integer. The disadvantage is that the ResultResponse is not re-used here.

Reply:
ok, good as is then.